generative_ai/text_embedding.py

```diff
@@ -0,0 +1,47 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START aiplatform_sdk_text_embedding]
+from google.cloud.aiplatform.private_preview.vision_models import ImageGenerationModel
```

Comment:
Error: `ModuleNotFoundError: No module named 'google.cloud.aiplatform.private_preview'`

Reply:
This need to be added to the allowlist. The SDK is not ready yet